docs/_layouts/docs.html

```diff
@@ -10,6 +10,20 @@
       <div class="unit four-fifths">
         <article>
           <h1>{{ page.title }}</h1>
+          {% assign importer = page.path | split: "/" | last | split: "." | first %}
+          {% assign importer_deps = site.data.importer_dependencies[importer] %}
+          {% if importer_deps %}
+          <div class="note info">
+            <h5>Additional Dependencies</h5>
```

Comment:
Could you paste a screenshot of how this looks?

Reply:
Viewport screenshort

![drupal_importer_deps](https://user-images.githubusercontent.com/12479464/156933484-46f99de7-0b89-420a-8009-de49a07c6e52.png)